deploy/travis/before_script

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euf -o pipefail
+
+if [ "$TRAVIS_JOB" = "test" ]; then
+    cp esp/esp/local_settings.py.travis esp/esp/local_settings.py
+    ln -s `pwd`/esp/public/media/default_images esp/public/media/images
+    ln -s `pwd`/esp/public/media/default_styles esp/public/media/styles
+    psql -c "CREATE ROLE testuser PASSWORD 'testpassword' LOGIN CREATEDB;" -U postgres
+    psql -c "CREATE DATABASE test_django OWNER testuser;" -U postgres
+elif [ "$TRAVIS_JOB" = "lint" ]; then
+    :
+else
+    echo "Unknown TRAVIS_JOB: $TRAVIS_JOB"
+    exit -1
```

Comment:
is `exit -1` valid?  I thought exit codes had to be in `[0, 128)` for normal exits, and `[128, 256)` (or perhaps `[-128, 0)` for signals.